src/u2f/u2f_app.c

```diff
@@ -29,6 +29,108 @@ static const app_t _apps[] = {
                   "\x59\x43\x79\x4e\x88\x4f\x3d\x02\x3a\x82\x29\xfd",
         .name = "Google",
     },
+    {
+        // sha256('https://bitbucket.org')
+        .app_id = "\x12\x74\x3b\x92\x12\x97\xb7\x7f\x11\x35\xe4\x1f\xde\xdd\x4a\x84\x6a\xfe\x82\xe1"
+                  "\xf3\x69\x32\xa9\x91\x2f\x3b\x0d\x8d\xfb\x7d\x0e",
+        .name = "Bitbucket",
+    },
+    {
+        // sha256('https://www.bitfinex.com')
+        .app_id = "\x30\x2f\xd5\xb4\x49\x2a\x07\xb9\xfe\xbb\x30\xe7\x32\x69\xec\xa5\x01\x20\x5c\xcf"
+                  "\xe0\xc2\x0b\xf7\xb4\x72\xfa\x2d\x31\xe2\x1e\x63",
+        .name = "Bitfinex",
+    },
+    {
+        // sha256('https://vault.bitwarden.com/app-id.json')
+        .app_id = "\xa3\x4d\x30\x9f\xfa\x28\xc1\x24\x14\xb8\xba\x6c\x07\xee\x1e\xfa\xe1\xa8\x5e\x8a"
+                  "\x04\x61\x48\x59\xa6\x7c\x04\x93\xb6\x95\x61\x90",
+        .name = "Bitwarden",
+    },
+    {
+        // sha256('https://www.dashlane.com')
+        .app_id = "\x68\x20\x19\x15\xd7\x4c\xb4\x2a\xf5\xb3\xcc\x5c\x95\xb9\x55\x3e\x3e\x3a\x83\xb4"
+                  "\xd2\xa9\x3b\x45\xfb\xad\xaa\x84\x69\xff\x8e\x6e",
+        .name = "Dashlane",
+    },
+    {
+        // sha256('https://www.dropbox.com/u2f-app-id.json')
+        .app_id = "\xc5\x0f\x8a\x7b\x70\x8e\x92\xf8\x2e\x7a\x50\xe2\xbd\xc5\x5d\x8f\xd9\x1a\x22\xfe"
+                  "\x6b\x29\xc0\xcd\xf7\x80\x55\x30\x84\x2a\xf5\x81",
+        .name = "Dropbox",
+    },
+    {
+        // sha256('https://api-9dcf9b83.duosecurity.com')
```

Comment:
This is weird. api-9dcf9b83 feels like a "random" identifier and https://api-9dcf9b83.duosecurity.com responds with 404 not found. Where is it from?

[trezor's fido entries](https://github.com/trezor/trezor-firmware/blob/65b0eae89df72eff1561d4de7221c8e34562acd4/common/defs/fido/duo.json) does list it but I wouldn't add it until we know whether it's actually owned by duosecurity and not some kind of their 3rd party app or a user.

Reply:
> FYI - That hostname will be unique for each Duo customer account.

(from https://github.com/github/SoftU2F/pull/32#issuecomment-332979298)

I would definitely remove this.

Reply:
Removed.